addons/web/static/src/webclient/settings_form_view/fields/upgrade_boolean_field.scss

```diff
@@ -0,0 +1,3 @@
+.o_field_upgrade_boolean {
+    @extend .o_field_boolean;
```

Comment:
Should probably use the additionalClasses property on the upgrade boolean field so that it gets the o_field_boolean class applied to it instead, `@extend` is very touchy because it can generate a lot of compiled css code so you should avoid it unless you have a very good reason to use it.

Reply:
Duly noted, not much familiar with the intricacies of the CSS.
Didn't know about `additionalClasses`, thanks for the input !